internal/pkg/service/cli/options/options.go

```diff
@@ -50,9 +50,9 @@ func New() *Options {
 	}
 }
 
-func (o *Options) Load(ctx context.Context, logger log.Logger, osEnvs *env.Map, fs filesystem.Fs, flags *pflag.FlagSet) error {
+func (o *Options) Load(ctx context.Context, _ log.Logger, osEnvs *env.Map, fs filesystem.Fs, flags *pflag.FlagSet) error {
 	// Load ENVs from OS and files
-	o.envs = o.loadEnvFiles(ctx, logger, osEnvs, fs)
+	o.envs = o.loadEnvFiles(ctx, log.NewNopLogger(), osEnvs, fs)
```

Comment:
There is currently ENVs loading twice, by the new code and deprecated `options` package, so I temporary disabled logs from the options package, to fix tests.